sdk/documentintelligence/ai-document-intelligence-rest/src/documentIntelligence.ts

```diff
@@ -63,5 +65,39 @@ export default function createClient(
     },
   });
 
+  client.pipeline.addPolicy({
+    name: "ResponseStreamToUint8ArrayPolicy",
+    sendRequest: async (req, next) => {
+      logger.info("ResponseStreamToUint8ArrayPolicy starts");
```

Comment:
do we need these log statements? This feels more `verbose` than `info`

Reply:
Seems way too chatty indeed per request.

Reply:
removed `starts` and `ends`, converted others to `verbose`